packages/common/locales/generate-locales-tool/bin/write-locale-files-to-dist.ts

```diff
@@ -32,19 +32,22 @@ function main(outputDir: string|undefined) {
   const extraLocaleDir = join(outputDir, 'extra');
   const globalLocaleDir = join(outputDir, 'global');
 
-  console.info(`Writing locales to: ${outputDir}`);
-
   // Generate locale files for all locales we have data for.
-  cldrData.availableLocales.forEach((localeData) => {
+  await Promise.all(cldrData.availableLocales.map(async (localeData) => {
```

Comment:
```suggestion
  await Promise.all(cldrData.availableLocales.flatMap(async (localeData) => {
```


If we use a flat map and in lines 42-45 we just return the unawaited promises then we can "parallelize" all of the file writes instead of doing promise chunks.